de.dlr.sc.virsat.model.extension.statemachines.test/src/de/dlr/sc/virsat/model/extension/statemachines/excel/importer/ExcelImporterTest.java

```diff
@@ -37,37 +37,33 @@
 import de.dlr.sc.virsat.model.extension.statemachines.model.StateMachine;
 import de.dlr.sc.virsat.model.extension.statemachines.model.Transition;
 
-
-
-
 /**
  * Test Case for Importing from Excel
- * @author bell_er
- *
  */
 public class ExcelImporterTest {
 	private static final String CONCEPT_ID_MACHINES = "de.dlr.sc.virsat.model.extension.statemachines";
 
-	private static final int FIVE = 5;
-	private static final int FOUR = 4;
-	private static final int EXPECTEDSTATECOUNT = 4;
-	private static final int EXPECTEDTRANSITIONCOUNT = 4;
-	
-	ABeanStructuralElementInstance ed;
+	private static final int STATE_NOT_EXISTING = 6;
+	private static final int STATE_WITHOUT_UUID = 5;
+	private static final int STATE_WITHOUT_NAME = 3;
+	private static final int EXPECTED_STATECOUNT = 4;
+	private static final int EXPECTED_TRANSITIONCOUNT = 4;
+
+	ABeanStructuralElementInstance aBeanSei;

```

Comment:
Let's maybe rename it to `elementDefinition` instead of `aBeanSei`?

Reply:
done